enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProvider.java

```diff
@@ -172,11 +172,9 @@ public PresenceCheckResult getResult(OwnerId id, SessionInfo sessionInfo) throws
                 final ClientErrorResponse clientErrorResponse = parseResponse(e.getResponse(), ClientErrorResponse.class);
                 if (ClientErrorResponse.ErrorEnum.INVALID_TOKEN.equals(clientErrorResponse.getError())) {
                     logger.warn("Invalid iProov token - reused token or validation called before verification, {}", id);
-                    result.setStatus(PresenceCheckStatus.IN_PROGRESS);
-                } else {
-                    result.setStatus(PresenceCheckStatus.FAILED);
-                    result.setErrorDetail(e.getResponse());
                 }
+                result.setStatus(PresenceCheckStatus.FAILED);
```

Comment:
The logic can be shared for if-else branches (see 4 lines below).

Reply:
👍